### PR TITLE
Sonata Hybrid 2021

### PR DESCRIPTION
**Pull request by Matt-Wash-Burn**
***** Hyundai Sonata Hybrid: Car port *****

**Checklist**
- [x] added to README
- [x] test route added to [test_routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/test/test_routes.py)
- [x] route with openpilot:
- [ ] route with stock system:

### CODE REVIEW

File: selfdrive/car/hyundai/carcontroller.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, visual_alert,
 
     # 20 Hz LFA MFA message
     if frame % 5 == 0 and self.car_fingerprint in [CAR.SONATA, CAR.PALISADE, CAR.IONIQ, CAR.KIA_NIRO_EV, CAR.KONA_EV,
-                                                   CAR.IONIQ_EV_2020, CAR.IONIQ_PHEV, CAR.KIA_CEED, CAR.KIA_SELTOS, CAR.ELANTRA_2021, CAR.ELANTRA_HEV_2021]:
+                                                   CAR.IONIQ_EV_2020, CAR.IONIQ_PHEV, CAR.KIA_CEED, CAR.KIA_SELTOS, CAR.ELANTRA_2021, CAR.ELANTRA_HEV_2021,CAR.SONATA_HYBRID]:
```
Comment (is343) [inline selfdrive/car/hyundai/carcontroller.py:81]: Instead of `HYBRID`, should we stick with the `HEV` convention that's used elsewhere?

**Review by is343 — COMMENTED** [review]
Hey, thanks for doing this. I was planning on adding my 2021 Sonata Hybrid Limited once my comma 3 came in, but it looks like you beat me to it. I'm glad this is getting added. 

I just had a comment regarding the current naming convention for hybrids. What do you think?

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -36,6 +36,7 @@ class CAR:
   SONATA_LF = "HYUNDAI SONATA 2019"
   PALISADE = "HYUNDAI PALISADE 2020"
   VELOSTER = "HYUNDAI VELOSTER 2019"
+  SONATA_HYBRID= "HYUNDAI SONATA HYBRID 2021"
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:39]:
```suggestion
  SONATA_HYBRID = "HYUNDAI SONATA HYBRID 2021"
```

File: selfdrive/car/hyundai/carcontroller.py (hunk 1 above)
Comment (adeebshihadeh) [reply selfdrive/car/hyundai/carcontroller.py:81]: I want to update them all to hybrid anyway to match the other makes

Comment (is343) [reply selfdrive/car/hyundai/carcontroller.py:81]: Ah alright, thanks. Figured we should keep it consistent. If we want to change the others instead, that makes sense too.

**Pull request merged**